### PR TITLE
[bitnami/rabbitmq] Allow both definitions and setting credentials

### PR DESCRIPTION
**Pull request by javsalgar**
Signed-off-by: Javier Salmeron Garcia <jsalmeron@vmware.com>

**Description of the change**

Up until now, the load definitions and the RabbitMQ credentials were incompatible. In the latest revision of the container we will allow setting both elements. If the "users" section inside the loadDefinitions file is set, then the credentials will be ignored.

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes https://github.com/bitnami/charts/issues/5976

**Additional information**

**Checklist**

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [ ] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [ ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/rabbitmq/templates/statefulset.yaml [hunk 1]
```diff
@@ -183,11 +183,12 @@ spec:
...
             - name: RABBITMQ_LOAD_DEFINITIONS
               value: "yes"
-            - name: RABBITMQ_SECURE_PASSWORD
-              value: "no"
+            - name: RABBITMQ_DEFINITIONS_FILE
+              value: {{ .Values.loadDefinition.file | quote }}
             {{- else }}
             - name: RABBITMQ_LOAD_DEFINITIONS
               value: "no"
+            {{- end }}
```
Comment (migruiz4) [inline bitnami/rabbitmq/templates/statefulset.yaml:191]:
```suggestion
            - name: RABBITMQ_LOAD_DEFINITIONS
              value: {{ ternary "yes" "no" .Values.loadDefinition.enabled | quote }}
            - name: RABBITMQ_DEFINITIONS_FILE
              value: {{ .Values.loadDefinition.file | quote }}
```

**Review by migruiz4 — CHANGES_REQUESTED** [review]
LGTM! Just a small suggestion to reduce the template logic

**Review by migruiz4 — APPROVED** [review]
LGTM!

**Review by migruiz4 — APPROVED** [review]

**Comment by javsalgar** [comment]
Hi! 

We just released a new revision of the chart. Now, in case there is a definition WITHOUT the `users` section, it will set the credentials in the chart secret. If the `users` section is set, the credentials will be ignored.

I think this should cover your use case! :D

**Pull request merged**